### ISSUE TITLE
react-native 0.69 support: need to update @react-native-community/blur

### ISSUE THREAD

**Issue by ryanliljestrom** [alessiocancian/react-native-actionsheet#20]
**Is your feature request related to a problem? Please describe.**

Currently running our react-native@0.68 app, we get console warnings like:

```
ViewPropTypes will be removed from React Native. Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types'
      at Object.ViewPropTypes (node_modules/react-native/index.js:470:13)
      at Object.<anonymous> (node_modules/@react-native-community/blur/src/BlurView.ios.js:28:7)
      at Object.<anonymous> (node_modules/@react-native-community/blur/index.js:1:1)
```

`ViewPropTypes` is currently deprecated, and is completely removed in react-native@0.69

**Describe the solution you'd like**

The version of `@react-native-community/blur` needs to be updated. The `4.0.0` release notes mentions a fix for this issue: https://github.com/Kureev/react-native-blur/releases/tag/v4.0.0

**Comment by hayate** [comment]
Confirming that when running `react-native@0.70.3`, the above turned into errors:
```
Error: Requiring module "node_modules/@alessiocancian/react-native-actionsheet/lib/index.js", 
which threw an exception: Invariant Violation: ViewPropTypes has been removed from React Native. 
Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types'.
```

**Comment by claudiozam** [comment]
quick fix: https://www.npmjs.com/package/npm-force-resolutions

**Comment by alessiocancian** [comment]
Sorry for the delay, fixed in v3.1.3
